src/scss/global.scss

```diff
@@ -236,6 +233,19 @@ button.btn {
   }
 }
 
+// ensure that fonts aren't restored to default
+input,
+button,
+select,
+optgroup,
+textarea {
+  font-family: inherit;
+  font-size: inherit;
+}
+button {
+  cursor: pointer;
+}
+
```

Comment:
Why ensure it?
This may require more explanation in the comment. Few questions that arise:

Isn't `inherit` already browser default?
How about `unset`?
What reset makes them not inherit anyway?

Reply:
was needed for the previous dropdown implementation, should be good now ead67ab

Reply:
these are the styles that are set if these globals are removed 
for some reason they don't match to the design ones
![image](https://user-images.githubusercontent.com/56254806/152140179-cf7b1dbf-40b3-444c-bc6f-052bac003e5c.png)